internal/appsec/waf.go

```diff
@@ -281,7 +281,7 @@ func newGRPCWAFEventListener(handle *wafHandle, addresses map[string]struct{}, t
 			internalRuntimeNs atomic.Uint64
 			nbTimeouts        atomic.Uint64
```

Comment:
I don't think we need these anymore now that we use a single WAF context, they should be accounted for automatically IIRC in every run of the context, leading to the same code as in the HTTP handler where we didn't need to do this (because single context).